python_bindings/src/PyBuffer.cpp

```diff
@@ -272,6 +276,13 @@ void define_buffer(py::module &m) {
             return Buffer<>(type, sizes, name);
         }), py::arg("type"), py::arg("sizes"), py::arg("name") = "")
 
+        // Note that this exists solely to allow you to create a Buffer with a null host ptr;
+        // this is necessary for some bounds-query operations (e.g. Func::infer_input_bounds).
+        // TODO: is this the best name?
+        .def_static("make_query", [](Type type, const std::vector<int> &sizes, const std::string &name) -> Buffer<> {
```

Comment:
I think I'd spell it out `make_bounds_query`.

Reply:
Done